src/vsr/client.zig

```diff
@@ -390,7 +390,7 @@ pub fn Client(comptime StateMachine_: type, comptime MessageBus: type) type {
             self.ping_timeout.reset();
 
             const ping = Header{
-                .command = .ping,
+                .command = .ping_client,
```

Comment:
Wait, this doesn't make sense. The command is `.ping_replica`, but we `send_header_to_replicas`. Either this should remain `.ping`, or I completely misunderstand the intended semantics here.

Reply:
My intent is:

- `ping_client` is client -> replica
- `pong_client` is replica -> client
- `ping` is replica -> replica
- `pong` is replica -> replica

So `_client` denotes that the client is on one end of the message, not necessarily that they are the receiver.

Reply:
:facepalm: I was thinking this is `replica.zig`, not `client.zig`.